docs/style_guide.md

```diff
@@ -36,7 +36,7 @@ The goal of coding standards is to make code easier to understand and maintain.
 6. You may use `/* ... */` or `//` for comments spanning multiple lines, just be consistent.
 
 
-# Functions
+## Functions
```

Comment:
Not sure if I completely agree with 5, but I guess style is subjective. In some situations (like the example here), for me it feels "cleaner" if we separate the "computing" and "returning".